cmd/tether/attach.go

```diff
@@ -419,9 +480,13 @@ func (t *attachServerSSH) run() error {
 
 				channel.Close()
 
-				serverConn.Lock()
-				serverConn.ServerConn = nil
-				serverConn.Unlock()
+				t.serverConn.Lock()
+				// set serverConn to nil if not an exec session
```

Comment:
this should explain what the desired effect is we accomplish through this.
I can see that we used to do this on for all session cleanup, but now that we have the code doing so in the global cleanup, is this needed/necessary?